lib/datadog/appsec/processor/rule_loader.rb

```diff
@@ -74,9 +74,6 @@ def passlist_exclusions(ip_passlist) # rubocop:disable Metrics/MethodLength
             when Hash
               pass = ip_passlist[:pass]
               monitor = ip_passlist[:monitor]
-            else
-              pass = []
-              monitor = []
```

Comment:
This is actually dead code, since `ip_passlist` comes from a non-nillable configuration option: it can be empty, but not `nil`.